tests/laravel-test.sh

```diff
@@ -2,31 +2,32 @@
 
 set -e
 
-LARAVEL_VERSION_CONSTRAINT="${1:-^9.0}"
+LARAVEL_VERSION_CONSTRAINT="${1:-^10}"
 
 echo "Install Laravel ${LARAVEL_VERSION_CONSTRAINT}"
-composer create-project --quiet --prefer-dist "laravel/laravel:${LARAVEL_VERSION_CONSTRAINT}" ../laravel
-cd ../laravel/
+rm -rf tests/laravel
+composer create-project --quiet --prefer-dist "laravel/laravel:${LARAVEL_VERSION_CONSTRAINT}" tests/laravel
+cd tests/laravel
 
 echo "Add Larastan from source"
 composer config minimum-stability dev
-composer config repositories.0 '{ "type": "path", "url": "../larastan", "options": { "symlink": false } }'
+composer config repositories.0 '{ "type": "path", "url": "../", "options": { "symlink": false } }'
 # No version information with "type":"path"
 composer require --dev "larastan/larastan:*"
-composer du -o
+composer du --optimize
 
 cat >phpstan.neon <<"EOF"
 includes:
-    - ./vendor/larastan/larastan/extension.neon
+    - vendor/larastan/larastan/extension.neon
 parameters:
     level: 5
     paths:
-        - app/
+        - app
```

Comment:
This a very good PR!
The slash here signifies that app is a directory. Please revert it.